content/docs/hello-world.md

```diff
@@ -15,36 +15,43 @@ ReactDOM.render(
+Van zodra je ze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.
 
 >Tip
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Deze gids is bedoeld voor zij die liever **concepten stap voor stap leren**. Als je liever al doende leert, bekijk dan onze [praktische tutorial](/tutorial/tutorial.html). Mogelijk vind je dan dat deze gids en de tutorial elkaar aanvullen.
 
-This is the first chapter in a step-by-step guide about main React concepts. You can find a list of all its chapters in the navigation sidebar. If you're reading this from a mobile device, you can access the navigation by pressing the button in the bottom right corner of your screen.
+Dit is het eerste hoofdstuk in een stap-voor-stapgids over de belangrijkste React-concepten.
+Je kan een lijst van alle hoofdstukken vinden in de navigatiezijbalk.
+Als je dit leest vanaf een mobiel apparaat, kan je de navigatie openen door op de knop in de rechterbenedenhoek van je scherm te drukken.
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Elk hoofdstuk in deze gids bouwt verder op de kennis die in voorgaande hoofdstukken werd opgebouwd.
+**Het grootste deel van React kan je leren door de hoofdstukken “Algemene Concepten” te lezen in de volgorde waarin ze verschijnen in de zijbalk**.
+[“JSX: Een Introductie”](/docs/introducing-jsx.html) is bijvoorbeeld het volgende hoofdstuk.
 
-## Knowledge Level Assumptions {#knowledge-level-assumptions}
+## Veronderstellingen Op Vlak Van Kennis {#knowledge-level-assumptions}
 
-React is a JavaScript library, and so we'll assume you have a basic understanding of the JavaScript language. **If you don't feel very confident, we recommend [going through a JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level** and enable you to follow along this guide without getting lost. It might take you between 30 minutes and an hour, but as a result you won't have to feel like you're learning both React and JavaScript at the same time.
+React is een JavaScript library en dus zullen we veronderstellen dat je een basisbegrip hebt van de JavaScript taal.
+**Als je hier niet erg zeker over bent, raden we [deze JavaScript tutorial](https://developer.mozilla.org/nl/docs/Web/JavaScript/A_re-introduction_to_JavaScript) aan om je niveau te testen** en om je in staat te stellen deze gids mee te volgen zonder te verdwalen.
```

Comment:
I think it's better to not directly reference the tutorial we're linking, just like the English version, as it causes an assumption that it's an official tutorial or that going through the tutorial will guarantee readers not having problems with React.
```suggestion
**Als je hier niet erg zeker over bent, raden we [een JavaScript tutorial](https://developer.mozilla.org/nl/docs/Web/JavaScript/A_re-introduction_to_JavaScript) aan om je niveau te testen** en om je in staat te stellen deze gids mee te volgen zonder te verdwalen.
```
```suggestion
**Als je hier niet erg zeker over bent, raden we [deze JavaScript tutorial](https://developer.mozilla.org/nl/docs/Web/JavaScript/A_re-introduction_to_JavaScript) aan om je niveau te testen** en om je in staat te stellen deze gids te volgen zonder te verdwalen.
```